paper.tex

```diff
@@ -248,7 +258,7 @@ \subsubsection{Numerical Example}
 
 \section{\normalsize\textbf{Future Work}}
 %Data Auction
-The data auction process is a data trading mechanism and an economically-driven scheme that establishes corresponding prices of data products through bidding process between consumers and providers. While there has been many auction models\cite{BigPicDataMarket} in several areas, only a few of research focus on third-party auction platform. Our proposed decentralized data marketplace protects the privacy of participants which reveals the minimum information for validation and reduces the suspicion of trust to centralized parties or auctioneers. However, the auction process between multiple participants and analyzing potential threats are still critical issues that need to be resolved.
+The data auction process is a data trading mechanism and an economically-driven scheme that establishes corresponding prices of data products through bidding process between consumers and providers. While there has been many auction models\cite{BigPicDataMarket} in several areas, only a few of research focus on third-party auction platform. Our proposed decentralized data marketplace protects the privacy of participants which reveals the minimum information for validation and reduces the suspicion of trust to centralized parties or auctioneers. However, the auction process between multiple participants and analysis of potential threats are still critical issues that need to be resolved. The game theory model presented in this work, we assume each data provider's behavior is independent. However, normally there would be multiple data providers provide similar product (substitution). Multiple providers must be taken account in the future works.
```

Comment:
Replace "Multiple providers must be taken account in the future works." with "Only one provider is taken into considerations at present."